docs/api-references/order.md

```diff
@@ -0,0 +1,53 @@
+## Order - API
+
+### `POST` Add New Order
+```
+api/orders/create
+```
+Add new order from client.
+#### Request Header
+| Key | Value |
+|---|---|
+| Accept | application/json |
+|Content-Type| application/json |
+| Authorization | {token_type} {access_token} |
+#### Parameters
+| Key | Type | Required | Description |
+|---|---|---|---|
+| user_id | Integer | required | Id of user |
```

Comment:
Please add more description. this `user_id` for the current user or another user?